controller/config.go

```diff
@@ -11,6 +11,7 @@ import (
 
```

Comment:
What's with the re-ordering of things in this file? I *think* nothing is removed, only added but it makes it quite hard to determine if this is more of an order/whitespace change or if anything is removed.

Reply:
Got a little carried away there. Fixed!